eng/pipelines/mgmt.yml

```diff
@@ -1,24 +1,277 @@
-# DO NOT EDIT THIS FILE
-# This file is generated automatically and any changes will be lost.
-
-trigger: none
-pr: none
-
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - sdk/advisor/Microsoft.Azure.Management.Advisor
```

Comment:
@isra-fel do you guys depend on automatic CI runs on every commit or do you just do a nightly or manual runs once changes are in? I ask because we could avoid having this list twice if we only needed it for PR validation and not CI triggering.

Reply:
Yeah we only need PR validation

Reply:
If we only add includes for the PR validations won't it mean that data-plane changes will still trigger this on CI?

Reply:
We should set CI trigger to none.

Reply:
I updated the script.